commit 44d03031a1af4d4d67dfccc2f9118ca82ae883e9
Author: Michael Cyr <mikecyr@us.ibm.com>
Date:   Fri Mar 17 00:48:20 2017 +0000

    scsi: ibmvscsis: Return correct partition name/# to client
    
    [ Upstream commit 9c93cf03d4eb3dc58931ff7cac0af9c344fe5e0b ]
    
    Signed-off-by: Michael Cyr <mikecyr@us.ibm.com>
    Signed-off-by: Bryant G. Ly <bryantly@linux.vnet.ibm.com>
    Tested-by: Steven Royer <seroyer@linux.vnet.ibm.com>
    Signed-off-by: Martin K. Petersen <martin.petersen@oracle.com>
    Signed-off-by: Sasha Levin <alexander.levin@verizon.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1417_44d03031a1af4d4d67dfccc2f9118ca82ae883e9.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1417_44d03031a1af4d4d67dfccc2f9118ca82ae883e9.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
